src/components/editor/prosemirror/module.ts

```diff
@@ -21,20 +23,39 @@ export { EditorView, EditorState, Plugin, defaultMarkdownSerializer };
 const schema = createSchema();
 const parser = createMarkdownParser(schema);
 
-export function createEditorState(opts: {content: ContentContainer }) {  
-    return EditorState.create({
-        doc: parser.parse(opts.content.textContent),
-        plugins: [
-            ...setupPlugins({ schema }),
-            new Plugin({
-                view: () => {
-                    return {
-                        update: debounce((view: EditorView) => {
-                            opts.content.textContent = defaultMarkdownSerializer.serialize(view.state.doc);
-                        }, 50)
-                    };
-                },
-            }),
-        ],
+export function createProseMirrorEditor(element: HTMLElement, cell: Cell, opts: { focusAfterInit?: boolean }, runtime: Runtime) {
+    let editorView = new EditorView(element, {
+        state: EditorState.create({
+            doc: parser.parse(cell.textContent),
+            plugins: [
+                keymap({
+                    "ArrowDown": function (state, dispatch, view) {
+                        if(state.selection.empty) {
+                            // Now what?
```

Comment:
Don't let this be a blocker for merging this PR, I'm happy to merge it without us figuring this out first - we can leave this stub code for now and figure out prosemirror later.